src/components/modals/AddDacModal.js

```diff
@@ -1,27 +1,42 @@
+import _ from 'lodash';
 import { Component } from 'react';
 import { div, form, h, hh, input, label, textarea } from 'react-hyperscript-helpers';
-import AsyncSelect from 'react-select/lib/Async';
+import AsyncSelect from 'react-select/async';
+import { DAC } from '../../libs/ajax';
 import { Alert } from '../Alert';
 import { BaseModal } from '../BaseModal';
-import { DAC } from '../../libs/ajax';
+import { DacUsers } from '../DacUsers';
+
 
 export const AddDacModal = hh(class AddDacModal extends Component {
 
   constructor(props) {
     super(props);
     this.state = {
       isEditMode: this.props.isEditMode,
-      dac: this.props.isEditMode ? this.props.dac : {},
+      dac: this.props.isEditMode ? this.props.dac : { dac: {}, chairpersons: [], members: [] },
```

Comment:
this.props.dac have same { dac: {}, chairpersons: [], members: [] } structure ?  So, there is a state.dac.dac element ?

Reply:
See previous response.